### PR TITLE
Update messages and logs to designs from patterns

### PR DESCRIPTION
**Pull request by Chadha93**
**Notes for Reviewers**
Update messages and logs to designs from patterns

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Chadha93** [comment]
Golang and MeshKit failing checks, due to changes in string response too? Is there a hardcoded string matching happening... @MUzairS15?

**Comment by MUzairS15** [comment]
@Chadha93 The checks are failing for the mesheryctl meshkit error utility.

File: server/internal/graphql/resolver/error.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func ErrKubectlDescribe(err error) error {
 }
 
 func ErrPatternsSubscription(err error) error {
-	return errors.New(ErrConfigurationPatternsCode, errors.Alert, []string{"Configuration Subscription failed", err.Error()}, []string{"GraphQL subscription for Patterns stopped"}, []string{"Could be a network issue"}, []string{"Confirm that Meshery Server is reachable from your browser."})
+	return errors.New(ErrConfigurationPatternsCode, errors.Alert, []string{"Configuration Subscription failed", err.Error()}, []string{"GraphQL subscription for design stopped"}, []string{"Could be a network issue"}, []string{"Confirm that Meshery Server is reachable from your browser."})
```
Comment (MUzairS15) [inline server/internal/graphql/resolver/error.go:111]:
```suggestion
	return errors.New(ErrConfigurationPatternsCode, errors.Alert, []string{"Configuration Subscription failed", err.Error()}, []string{"GraphQL subscription for designs stopped"}, []string{"Could be a network issue"}, []string{"Confirm that Meshery Server is reachable from your browser."})
```

File: server/models/remote_provider.go [hunk 2]
```diff
@@ -1536,7 +1536,7 @@ func (l *RemoteProvider) GetMesheryPatterns(tokenString string, page, pageSize,
 
 	ep, _ := l.Capabilities.GetEndpointForFeature(PersistMesheryPatterns)
 
-	logrus.Infof("attempting to fetch patterns from cloud")
+	logrus.Infof("attempting to fetch design from cloud")
```
Comment (MUzairS15) [inline server/models/remote_provider.go:1539]:
```suggestion
	logrus.Infof("attempting to fetch designs from cloud")
```

File: server/models/remote_provider.go [hunk 3]
```diff
@@ -1556,15 +1556,15 @@ func (l *RemoteProvider) GetMesheryPatterns(tokenString string, page, pageSize,
...
+	logrus.Debugf("constructed design url: %s", remoteProviderURL.String())
 	cReq, _ := http.NewRequest(http.MethodGet, remoteProviderURL.String(), nil)
 
 	resp, err := l.DoRequest(cReq, tokenString)
 	if err != nil {
 		if resp == nil {
 			return nil, ErrUnreachableRemoteProvider(err)
 		}
-		logrus.Errorf("unable to get patterns: %v", err)
+		logrus.Errorf("unable to get design: %v", err)
```
Comment (MUzairS15) [inline server/models/remote_provider.go:1567]:
```suggestion
		logrus.Errorf("unable to get designs: %v", err)
```

File: server/models/remote_provider.go [hunk 4]
```diff
@@ -1577,11 +1577,11 @@ func (l *RemoteProvider) GetMesheryPatterns(tokenString string, page, pageSize,
 	}
 
 	if resp.StatusCode == http.StatusOK {
-		logrus.Infof("patterns successfully retrieved from remote provider")
+		logrus.Infof("design successfully retrieved from remote provider")
```
Comment (MUzairS15) [inline server/models/remote_provider.go:1580]:
```suggestion
		logrus.Infof("designs successfully retrieved from remote provider")
```

File: server/models/remote_provider.go [hunk 5]
```diff
@@ -1577,11 +1577,11 @@ func (l *RemoteProvider) GetMesheryPatterns(tokenString string, page, pageSize,
...
 		return bdr, nil
 	}
-	logrus.Errorf("error while fetching patterns: %s", bdr)
-	return nil, fmt.Errorf("error while fetching patterns - Status code: %d, Body: %s", resp.StatusCode, bdr)
+	logrus.Errorf("error while fetching design: %s", bdr)
```
Comment (MUzairS15) [inline server/models/remote_provider.go:1583]:
```suggestion
	logrus.Errorf("error while fetching designs: %s", bdr)
```

File: server/models/remote_provider.go [hunk 6]
```diff
@@ -1593,7 +1593,7 @@ func (l *RemoteProvider) GetCatalogMesheryPatterns(tokenString string, page, pag
 
 	ep, _ := l.Capabilities.GetEndpointForFeature(MesheryPatternsCatalog)
 
-	logrus.Infof("attempting to fetch catalog patterns from cloud")
+	logrus.Infof("attempting to fetch catalog design from cloud")
```
Comment (MUzairS15) [inline server/models/remote_provider.go:1596]:
```suggestion
	logrus.Infof("attempting to fetch catalog designs from cloud")
```

**Review by MUzairS15 — APPROVED** [review]

**Review by leecalcote — APPROVED** [review]
LGTM

**Review by Chadha93 — COMMENTED** [review]
Let's merge, thanks @MUzairS15 for making some tweaks.

**Pull request merged**